specificatie/genereervariant/openapi.yaml

```diff
@@ -3270,8 +3299,8 @@ components:
         minItems: 2
         type: array
         example:
-        - 196733.51
-        - 439931.89
+        - "196733.510"
```

Comment:
Zelfde fout als bij de locatie-query op regel 2009. Dit moeten nummers zijn, niet nummers in string